app/[lang]/not-found.tsx

```diff
@@ -0,0 +1,37 @@
+import '../../styles/globals.css'
```

Comment:
We will need to add a line to make sure this renders on the client side if we want to have this page be translatable 
``` suggestion
'use client'

```